src/Chapter2.hs

```diff
@@ -620,7 +665,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven (x:xx:xs) = [x] ++ takeEven xs
```

Comment:
As you don't use `xx` at the right part, you can safely use a wildcard instead.
And also, remember that `:` is muuuch more efficient 😉 
```suggestion
takeEven (x:_:xs) = x : takeEven xs
```